scripts/observability/user.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+#######################################
+# Login to Keycloak and get an access token
+# Globals:
+#  SSO_ACCESS_TOKEN
+# Arguments:
+#   kc_url (Keycloak)
+#   kc_realm (Keycloak)
+#   kc_client_id (Keycloak)
+#   kc_client_secret (Keycloak)
+#   kc_username (Keycloak)
+#   kc_password (Keycloak)
+# Examples:
+#   observability_keycloak_login https://sso.suse.com instruqt suse xxxxxx admin password
+#######################################
```

Comment:
Doesn't seem restricted to observability and we could use this elsewhere
Could you create an authentication folder with a keycloak script in it with all Keycloak functions?